rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,68 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+# the lines below are sed'ed by build-server-rpm script to set a correct version
+# see: https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+%global commit            0000000000000000000000000000000000000000
+%define full_pmm_version 2.0.0
+
+%global shortcommit       %(c=%{commit}; echo ${c:0:7})
+%define build_timestamp   %(date -u +"%y%m%d%H%M")
+%define release           2
+%define rpm_release       %{release}.%{build_timestamp}.%{shortcommit}%{?dist}
+
+%global install_golang    0
+
+Name:       dbaas-controller
+# the line below is sed'ed by build-server-rpm script to set a correct version
+# see: https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+Version:    %{version}
+Release:    %{rpm_release}
+Summary:    Simplified API for managing Percona Kubernetes Operators
+
+License:    AGPLv3
+URL:        https://github.com/percona-platform/dbaas-controller
+Source0:    https://github.com/percona-platform/dbaas-controller/archive/%{commit}/dbaas-controller-%{shortcommit}.tar.gz
+
+%if %{install_golang}
+BuildRequires:   golang >= 1.14.0
+%endif
+
+%description
+dbaas-controller exposes a simplified API for managing Percona Kubernetes Operators
+See the PMM docs for more information.
+
+
+%prep
+%setup -q -n dbaas-controller-%{commit}
+
+
+%build
+export PMM_RELEASE_VERSION=%{full_pmm_version}
+export PMM_RELEASE_FULLCOMMIT=%{commit}
+export PMM_RELEASE_BRANCH=""
+
+make release \
+ PMM_RELEASE_VERSION=%{full_pmm_version} \
+ PMM_RELEASE_FULLCOMMIT=%{commit} \
+ PMM_RELEASE_BRANCH=""
```

Comment:
We already export them above. They will be used if everything else is set correctly. That line should be just `make release`: https://github.com/percona/pmm-server-packaging/blob/064493f56992516f23b27b45bc93c9e4734a84e2/rhel/SPECS/pmm-managed.spec#L52-L57

Please do not add hacks.

Reply:
Fixed